website/static/js/pages/base-page.js

```diff
@@ -60,6 +60,17 @@ var NO_FOOTER_PATHS = ['/login/', '/getting-started/', '/register/'];
 if ($(sliderSelector).length > 0 &&
         $.inArray(window.location.pathname, NO_FOOTER_PATHS) === -1) {
     $osf.applyBindings(new SlideInViewModel(), sliderSelector);
+
+    // Click event bindings for Google Analytics
+    $('#footerSlideInLoginBtn').on('click', function() {
```

Comment:
Instead of relying on CSS IDs for event handling, you could add these handlers to the existing `SlideInViewModel`.